docs/release-notes/2.38.1.md

```diff
@@ -0,0 +1,11 @@
+# Percona Monitoring and Management 2.38.1
+
+| **Release date:** | July 11, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) With this update, we are fixing an authentication issue that was causing PMM to incorrectly display a "User already exists" error when trying to re-login into PMM using a Percona Account.
```

Comment:
The error is no a problem. the problem - inability to login with PErcona Account

Reply:
@rnovikovP should we also mention how to fix it for instances with other identity providers affected?

Reply:
yes, we need to mention this 100%